angular/core/components/group_page/join_group_button/join_group_button.coffee

```diff
@@ -1,13 +1,13 @@
 angular.module('loomioApp').directive 'joinGroupButton', ->
-  scope: {group: '='}
+  scope: {group: '=', block: '=?'}
```

Comment:
Ack sorry, I'm jumping to conclusions again. We _do_ need this for when you're

- logged in
- viewing a public discussion in a group you're not a member of.

That's when you get this:
![screen shot 2016-05-10 at 10 32 09 am](https://cloud.githubusercontent.com/assets/750477/15133062/7c9d09a8-169a-11e6-8baf-48ea3f532c19.png)